platform/linuxbsd/wayland/wayland_thread.cpp

```diff
@@ -1893,7 +1893,7 @@ void WaylandThread::_wl_pointer_on_frame(void *data, struct wl_pointer *wl_point
 					pd.last_pressed_position = pd.position;
 				}
 
-				if (old_pd.double_click_begun && mb->is_pressed() && pd.last_button_pressed == old_pd.last_button_pressed && (pd.button_time - old_pd.button_time) < 400 && Vector2(old_pd.last_pressed_position * scale).distance_to(Vector2(pd.last_pressed_position * scale)) < 5) {
+				if (old_pd.double_click_begun && mb->is_pressed() && pd.last_button_pressed == old_pd.last_button_pressed && (pd.button_time - old_pd.button_time) < 400 && Vector2(old_pd.last_pressed_position * scale).distance_squared_to(Vector2(pd.last_pressed_position * scale)) < 25.0) {
```

Comment:
At this point I realized I've reviewed half the PR and have been writing way too many of these comments, I'll stop for now and look over things again. There are more cases of things that need to be `f` suffixed.

```suggestion
				if (old_pd.double_click_begun && mb->is_pressed() && pd.last_button_pressed == old_pd.last_button_pressed && (pd.button_time - old_pd.button_time) < 400 && Vector2(old_pd.last_pressed_position * scale).distance_squared_to(Vector2(pd.last_pressed_position * scale)) < 25.0f) {
```